python/raydp/spark/ray_cluster_master.py

```diff
@@ -60,6 +60,10 @@ def start_up(self, popen_kwargs=None):
 
     def _prepare_jvm_classpath(self):
         cp_list = []
+
+        if 'executor.worker.extraClassPath' in self._configs:
```

Comment:
Please use " instead of ' to fix the lint error. Besides, I think using raydp.executor.extraClassPath is better. And we should update the doc to explain this

Reply:
_Please use " instead of ' to fix the lint error._ 

What does it mean? I'm a little bit confused

Reply:
Got it. I have changed it and added the doc